misc/python/materialize/mzcompose.py

```diff
@@ -877,6 +878,56 @@ def __init__(
         super().__init__(name=name, config=config)
 
 
+class Redpanda(PythonService):
+    def __init__(
+        self,
+        name: str = "redpanda",
+        version: str = "v21.10.2",
```

Comment:
@benesch I bumped the Redpanda version here. I know you did not want any jobs in the CI that reference `latest`, but, in the spirit of cooperation, we should be testing their stuff as soon as it lands on dockerhub.

Reply:
FWIW, I think it's fine, possibly even desirable, to have nightly tests run against `latest`! I just don't want anything that runs against a PR or a release build to depend on nonreproducible state. Otherwise random PRs or, worse, the Materialize release process can get wedged by bugs introduced in a new version of Redpanda.

Reply:
Thank you, will open a follow-up task for this.